.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+A test.md
```

Comment:
[ Optional ]
 test.md  if ignored should not be pushed to github , I suggest next time to add file name to ignore to gitignore like this /test.md  before you create it.